src/contracts/BBFreelancerPayment.sol

```diff
@@ -81,9 +81,9 @@ contract BBFreelancerPayment is BBFreelancer{
   * @dev check payment status 
   **/
   function checkPayment(bytes jobHash) public view returns(uint256, uint256){
-    uint256 finishDate = bbs.getUint(keccak256(abi.encodePacked(jobHash,'JOB_FINISHED_TIMESTAMP')));
-    uint256 paymentLimitTimestamp = bbs.getUint(keccak256('PAYMENT_LIMIT_TIMESTAMP'));
-    uint256 status = bbs.getUint(keccak256(abi.encodePacked(jobHash,'STATUS')));
+    uint256 finishDate = bbs.getUint(keccak256(abi.encodePacked(jobHash,'finishedTimestamp')));
+    uint256 paymentLimitTimestamp = bbs.getUint(keccak256('PaymentLimitTimestamp'));
```

Comment:
Hey @chris-tn can you review naming convention again ? First letter of a variable should be a non-capitalized letter ?

Reply:
hey @onimisha i already change all keys name to capitalized letter from pr #34

this PR fixed for old key

Reply:
closed ! becasue the PR #34